packages/model-viewer/src/test/features/scene-graph/model-spec.ts

```diff
@@ -74,5 +79,58 @@ suite('scene-graph/model', () => {
 
       expect(collectedMaterials.size).to.be.equal(materials.size);
     });
+
+    suite('Model Variants', () => {
+      test('Switch variant and lazy load', async () => {
+        const threeGLTF = await loadThreeGLTF(SHEEN_CHAIR_GLB_PATH);
+        const model = new Model(CorrelatedSceneGraph.from(threeGLTF));
+
+        expect(model[$materials][4][$correlatedObjects]).to.be.null;
+        expect(model[$materials][4][$lazyLoadGLTFInfo]).to.not.be.undefined;
```

Comment:
I think there's a `.to.be.okay` or something like that. It would be a shame to get a null here and have the test pass.

Reply:
Okay.